include/services/internal/dynamic_lib_helper.h

```diff
@@ -73,6 +93,15 @@ class DynamicLibHelper final
             services::internal::tryAssignStatusAndThrow(status, services::ErrorCanNotLoadDynamicLibrarySymbol);
             return nullptr;
         }
+    #elif defined(_WIN32) || defined(_WIN64)
+        void * sym = GetProcAddress((HMODULE)_handle, symName);
+
+        if (nullptr != sym)
+        {
+            services::internal::tryAssignStatusAndThrow(status, services::ErrorCanNotLoadDynamicLibrarySymbol);
+            return nullptr;
```

Comment:
Suggest to replace with `NULL`, since we are not aware about C++11 support in header file that might include this one

Reply:
The same for line 94

Reply:
replaced with NULL